sycl/source/detail/kernel_info.hpp

```diff
@@ -120,55 +120,30 @@ get_kernel_work_group_info_host<info::kernel_work_group::private_mem_size>(
 }
 // The kernel sub-group methods
 
-template <typename TOut, info::kernel_sub_group Param>
-struct get_kernel_sub_group_info {
-  static TOut get(RT::PiKernel Kernel, RT::PiDevice Device,
-                  const plugin &Plugin) {
-    TOut Result;
+template <info::kernel_sub_group Param> struct get_kernel_sub_group_info {
+  static uint32_t get(RT::PiKernel Kernel, RT::PiDevice Device,
+                      const plugin &Plugin) {
+    uint32_t Result;
     // TODO catch an exception and put it to list of asynchronous exceptions
```

Comment:
```suggestion

```

Reply:
Should this comment be removed?

Reply:
AFAIK, this comment is not valid anymore. Most of the errors now reported through synchronous exceptions.
Although, I'm not the owner of the runtime and let @intel/llvm-reviewers-runtime to decide.